src/sentry/static/sentry/app/views/releases/list/organizationReleases/index.jsx

```diff
@@ -100,34 +86,42 @@ class OrganizationReleases extends React.Component {
     );
   }
 
-  render() {
-    if (!new Set(this.props.organization.features).has('sentry10')) {
-      return this.renderNoAccess();
-    }
+  renderError() {
+    return this.renderBody();
```

Comment:
Wouldn't we want to show the default error instead of body?

Reply:
Yes, good point